src/components/Landing/EventCard/EventCard.jsx

```diff
@@ -1,70 +1,75 @@
-import React, { useState, useEffect, useMemo } from "react";
+import React, { useState, useEffect } from "react";
 import { NavLink } from "react-router-dom";
 import PropTypes from "prop-types";
 import { BiRightArrowAlt } from "react-icons/bi";
 import moment from "moment";
-import IconsEventCard from "./IconsEventCard/IconsEventCard";
 import "./EventCard.scss";
 import PhotoService from "../../../services/PhotoService";
-import {EVENT_DEFAULT_IMAGE, FORMATS} from "../../../constants/eventConstants";
+import {
+  EVENT_DEFAULT_IMAGE,
+  FORMATS,
+} from "../../../constants/eventConstants";
 
-const EventCard = ({ event }) => {
+const EventCard = ({ event, additionalButtons, additionalModal }) => {
   const { id, title, dateFrom } = event;
-  const photoService = useMemo(() => new PhotoService(), []);
+  const photoService = new PhotoService();
   const [eventImage, setEventImage] = useState(EVENT_DEFAULT_IMAGE);
-
+  const sliceLength = title.length > 35 ? `${title.slice(0, 35)}...` : title;
```

Comment:
it can be rewrittern with css only

Reply:
@Danyil-man need move to utils or replace to scss code, if that not your task, revert all changes from this file